finalPrep/modZeroHero.js

```diff
@@ -56,29 +110,72 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+    name: "Stay Puft Marshmallow Man",
+    smell: "Mildly sweet",
+    weight: 7500,
+    citiesDestroyed: ["New York", "Chicago"],
+    luckyNumbers: [1, 7, 13]
+};
+    var address = {
+      number: "1984",
+      street: "Smith Rd",
+      state: "Alabama",
+      zip: "00001"
+};
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
 
+  sayName() {
+    this.name = "Igor";
```

Comment:
Running this function reassigns the value of this attribute. But how would you print the value of that attribute to the console?